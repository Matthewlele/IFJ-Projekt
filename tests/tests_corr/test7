string = "dreveny\sstul"
x = 567

print (length(string))
print (length(x))
