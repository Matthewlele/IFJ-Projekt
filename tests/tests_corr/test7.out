SYNTAX ERR: row 3
